packages/@aws-cdk/custom-resources/lib/aws-custom-resource/aws-custom-resource.ts

```diff
@@ -190,9 +190,21 @@ export interface AwsCustomResourceProps {
  *
  */
 export class AwsCustomResource extends cdk.Construct implements iam.IGrantable {
+
+  private static breakIgnoreErrorsCircuit(sdkCalls: Array<AwsSdkCall | undefined>, caller: string) {
```

Comment:
personal taste... why not a non exported function outside the class for this?

Reply:
I just prefer starting with the minimal visibility necessary. The function is used to validate specific configuration that can (currently) only arise in the `AwsCustomResource` class. I'd like additional usages of it to be very intentional and not just treat it as a sort of utility function.